Apps/StressTest/StressTest/ViewController.swift

```diff
@@ -79,8 +79,8 @@ class ViewController: UIViewController {
         annotations.reserveCapacity(100)
 
         // Do any additional setup after loading the view.
-        resourceOptions = ResourceOptions(accessToken: AccountManager.shared.accessToken!)
-        mapView = MapView(frame: view.bounds, resourceOptions: resourceOptions, styleURI: .streets)
+        mapInitOptions = MapInitOptions()
+        mapView = MapView(frame: view.bounds, mapInitOptions: mapInitOptions, styleURI: .streets)
```

Comment:
So the intial styleUri is passed as a separate parameter to the MapView, I think we could move it to the mapInitOptions, wdyt? cc @Chaoba 

Android has a [PR](https://github.com/mapbox/mapbox-maps-android/pull/248/files) to add the default style uri, we should align on this.

Reply:
We are loading the styleUri from mapboxMap, we might also need to align how to load it.

Reply:
FWIW I was going to treat the style uri handling as a separate PR - so we still have opportunity to discuss this.